src/rad/appdata.cpp

```diff
@@ -2391,8 +2392,9 @@ void ApplicationData::GenerateInheritedClass( PObjectBase form, wxString classNa
 		PProperty fileProp = obj->GetProperty( wxT( "file" ) );
 		PProperty genfileProp = obj->GetProperty( wxT( "gen_file" ) );
 		PProperty typeProp = obj->GetProperty( wxT( "type" ) );
+		PProperty typePCH = obj->GetProperty(wxT("precompiled_header"));
```

Comment:
Following the present naming style `pchProp` makes more sense and looks less confusing